src/libs/array-common.js

```diff
@@ -0,0 +1,17 @@
+export function pickNRandom(arr, n) {
```

Comment:
what if we create `libs/common/<mutation|array>` then indexed in `libs/common/index.js` ?
So that we could also called it in root modules?

Reply:
good idea, I'll do that. Thanks for the suggestion!